modules/standard/BigInteger.chpl

```diff
@@ -2492,6 +2499,19 @@ When ``numer/denom`` does not produce an integer, this method may produce incorr
       return false;
   }
 
+  /*
+    Return ``true`` if ``this`` is exactly divisible by ``div``.  ``this`` is
+    divisible by ``div`` if there exists an integer ``q`` satisfying ``this =
+    q*div``.  Unlike the other division functions, ``div = 0`` is accepted and
```

Comment:
In comments like this, I think it can be confusing to say something like `div = 0` or `div == 0` because they are overloaded terms, especially in this math context. I don't mind how it is, but I might change it to say something more like "0 is an acceptable value for `div` or something.

Reply:
How's the update look?

Reply:
Looks good!